frontend/pages/doctor/patientinfo.tsx

```diff
@@ -0,0 +1,77 @@
+import PatientDetailsToProvideForm from '@frontend/components/forms/PatientDetailsToProvideForm';
+import { Box, Flex, Heading, Text, Image, Divider } from '@chakra-ui/react';
+import { serverURL } from '@frontend/config/index';
+
+export async function getServerSideProps() {
+  let requiredDetails: any = [];
+  try {
+    // <TODO> Get current patient ID (the one that gets clicked on)
+    const response: any = await fetch(serverURL + '/patients/getRequiredDetails/1');
+    requiredDetails = await response.json();
+  } catch {}
```

Comment:
what happens if you hit the catch branch? Does it still return an empty array?

Reply:
Yep